index.html

```diff
@@ -41,6 +41,6 @@
         ga('create', gaID, 'auto');
         ga('send', 'pageview');
       </script>
-
+      <script src="https://unpkg.com/delayed-scroll-restoration-polyfill@0.1.1/index.js"></script>
```

Comment:
Polyfills should be installed via NPM and setup in the `clientEntry.js` file.

Reply:
yess added @brenoferreira